.travis.yml

```diff
@@ -19,17 +22,27 @@ env:
   - VERSION=8.x-1.01
   - VERSION=8.x-1.00
 
-sudo: false
-
-mysql:
-  database: drupal
-  username: root
-  encoding: utf8
+addons:
+  apt:
+    packages:
+    - mysql-server-5.6
+    - mysql-client-core-5.6
+    - mysql-client-5.6
 
 before_install:
   - phpenv config-rm xdebug.ini
   - composer self-update
   - composer --version
+  # Initialize xvfb (see https://docs.travis-ci.com/user/gui-and-headless-browsers)
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - sleep 3
+  - LATEST_CHROMEDRIVER=$(wget -q -O - http://chromedriver.storage.googleapis.com/LATEST_RELEASE)
+  - wget http://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER/chromedriver_linux64.zip
+  - unzip chromedriver_linux64.zip
+  - chmod +x chromedriver
+  - mkdir -p $HOME/.composer/vendor/bin
+  - mv -f chromedriver $HOME/.composer/vendor/bin/
```

Comment:
Do we need to start chromedriver or anything, or does it just need to be in the PATH?